src/sysext/sysext.c

```diff
@@ -1454,7 +1461,6 @@ static int store_info_in_meta(
                 const char *work_dir,
                 const char *hierarchy) {
         _cleanup_free_ char *f = NULL;
-        _cleanup_close_ int atfd = -EBADF;
         int r;
```

Comment:
We usually insert an empty line after multiline function prototype.
Also, move declaration of `f` to around `path_join()` below?